reindexer/reindex_request_processor/docker-compose.yml

```diff
@@ -1,3 +1,7 @@
+sns:
+  image: wellcome/fake-sns
+  ports:
+    - "9292:9292"
```

Comment:
Why does this application need SNS? It isn't sending notifications, is it?

Reply:
aha, thanks I got confused by the NotificationMessage on SQS